env_support/cmake/version.cmake

```diff
@@ -0,0 +1,6 @@
+set(LVGL_VERSION_MAJOR 8)
+set(LVGL_VERSION_MINOR 0)
+set(LVGL_VERSION_PATCH 3)
+set(LVGL_VERSION_INFO "dev")
```

Comment:
can we do this similar thing like: https://github.com/lvgl/lvgl/blob/master/scripts/find_version.sh
or call find_version.sh directly to simplify the release process.

Reply:
I don't know bash, but I guess `$LVGL_VERSION_MAJOR`, `$LVGL_VERSION_MINOR`, etc can be used to automatically create the content `version.cmake` file?

Reply:
Agree, even this commit has a bug, see MINOR version set in the proposed `version.cmake` file. @kisvegabor @embeddedt where do you think we should auto generate the `version.cmake` file?